pkgs/applications/audio/pamixer/default.nix

```diff
@@ -1,20 +1,30 @@
-{ lib, stdenv, fetchFromGitHub, fetchpatch, boost, libpulseaudio }:
+{ lib, stdenv, fetchFromGitHub, fetchpatch, boost, libpulseaudio, installShellFiles }:
 
 stdenv.mkDerivation rec {
   pname = "pamixer";
-  version = "1.4";
+  version = "unstable-2021-03-29";
```

Comment:
currently having a version starting with unstable messes up `builtins.parseDrvName` and  `nix run`
my recommendation is to either set `meta.mainProgram` (what i would probably do) or use some other style of version, which there is an ongoing discussion about normalization: https://github.com/NixOS/rfcs/pull/107

Reply:
Done.